packages/pwa-kit-extension-sdk/src/configs/webpack/override-stats-plugin.ts

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
I see why we can't combine this logic, one being a loader and one being a plugin. So I think it's goos the way it is.